vehicle/src/main/java/Vehicle.java

```diff
@@ -0,0 +1,62 @@
+import java.util.Date;
+
+/**
+ * Created by Michele on 12/07/2017.
+ */
+public class Vehicle {
+    private String carLicencePlate;
+    private String state; //can be: available, broken, recharging, booked
```

Comment:
I think that this can be actually an enum like:
```

public enum Status {
    AVAILABLE("available"),
    BROKEN("Broken")
    ;

    private final String value;

    /**
     * @param value
     */
    private Strings(String value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return text;
    }
}
```